README.md

```diff
@@ -7,46 +7,81 @@ to kick things off based on an existing skupper-site ConfigMap.
 # Installing the skupper-operator
 
 
-The instructions below demonstrates how to install the skupper-operator
-and run it inside the `my-namespace` namespace. If you want to install it in
+The instructions below demonstrate how to install the skupper-operator
+and run it inside the `my-namespace` namespace. If you want to install it on
 a different namespace, edit the referenced yaml before applying.
 
 If you want to test your catalog against a local minikube cluster,
 you'll need to install OLM first. For more info, check this out:
 https://olm.operatorframework.io/docs/getting-started/
 
-In an OpenShift cluster, OLM is already installed. So you just need to 
-create your CatalogSource.
+In an OpenShift cluster, OLM is already installed.
 
+# Installing from OperatorHub.io (Community Operators)
+
+Skupper Operator is also available on [OperatorHub.io](https://operatorhub.io/operator/skupper-operator).
+
+If the `Community Operators` catalog is available in your cluster, you
+can install the Skupper Operator by visiting the link above and clicking the `Install` button,
```

Comment:
Given that the `Install` button only displays doc and that doc only repeats what's below, but adds  mentions  of CRDs, maybe best to not draw attention to 'Install' button?